derive/src/to_delegate.rs

```diff
@@ -199,7 +218,7 @@ impl RpcMethod {
 			} else if param_types.is_empty() {
 				quote! { let params = params.expect_no_params(); }
 			} else if self.attr.raw_params {
-				quote! { let params = Ok((params,)); }
+				quote! { let params: _jsonrpc_core::Result<_> = Ok((params,)); }
```

Comment:
```suggestion
				quote! { let params = _jsonrpc_core::Result::Ok((params,)); }
```

Reply:
This sadly won't compile. We need a type annotation here for this to work in all cases.

Reply:
Oh, yeah. You're right. Sorry!